### DISCUSSION TITLE
Move/Copy Signal connections to other signals.

### DISCUSSION THREAD

**Discussion by KC098** [godotengine/godot-proposals#5227] [Editor]
Signals are a huge part of Godot workflow, so why not make it a bit easier to manage them by being able to just copy the signal connection/s to another or drag and drop them to another?

https://user-images.githubusercontent.com/73260374/185994980-2e33d6de-acd8-444f-8ab7-ce8675fce28e.mp4

**Comment by Calinou** [comment]
Out of curiosity, what would be some *concrete* use cases for this? In other words, in which scenarios do you need to assign a different signal to run the *same* function?

**Reply by KC098** [reply]
An example would be, say I want to call like 5-7 connections on a "visibility_change" signal, but then I want to change it to "child_entered_node" signal, I can just copy and paste it to that signal instead of reassigning the connections and disconnecting it one by one.

**Comment by azur-wolve** [comment]
+1
its rare that Godot already doesn't have such functionality IMO

**Reply by KC098** [reply]
Agree, I tried drag and dropping out of intuition to another signal and was surprised when that functionality wasn't implemented. Hopefully this gets added.
